.changeset/popular-eagles-sniff.md

```diff
@@ -0,0 +1,22 @@
+---
+"@react-email/render": major
+"react-email": patch
+---
+
+Deprecated `renderAsync` and made `render` itself always async
+
+## Why
+
+Three reasons:
+
+1. Better support of NextJS's latest versions
+2. Being ready for future React API deprecations
+3. Support for Suspense which allows for using async inside components
+
+See https://github.com/resend/react-email/discussions/1144 for more info.
+
+## How to upgrade
+
+You just need to treat the promise returned from `render` properly
```

Comment:
```suggestion
You need to treat the promise returned from `render` properly
```

let's not use "just", "simply" or similar words since we cannot assume the instructions are going to be "easy" for everyone 😄

Reply:
Changed the text up a bit to still be fluid but removed the `just`s